src/vcsparser.core/ChangesetProcessor.cs

```diff
@@ -7,22 +7,29 @@
 
 namespace vcsparser.core
 {
-    public class ChangesetProcessor
+    public interface IChangesetProcessor

```

Comment:
Please, extract to a new .cs file.

Reply:
Moved to new file in aa56194610a50f859ae3ccf7dd0d28b7c945eb4a